src/main/frontend/components/query_table.cljs

```diff
@@ -135,7 +135,7 @@
                                  (:block/pre-block? row)
                                  content)]
       (if (seq title)
-        [:element (->elem :div (map-inline config title))]
+        [:element (->elem :div (map-inline row title))]
```

Comment:
@xyhp915 What's the purpose of this change?

Reply:
This `config` is not the current row-based block config, so the uuid pointing to the row render is not correct.

- ref #10021